docs/modules/export.md

```diff
@@ -21,6 +21,16 @@ REST controller responsible for handling export operations related to templates.
 [export-api-v1.yaml](../api-specs/export-api-v1.yaml)
 
 ## Setup
+This module has to be enabled through a feature flag in the environment:
+
+in .env file:
+`FEATURES_EXPORTAPI_ENABLED=true`
+or as .yml:
+```
+features:
+  export-api:
+    enabled: true
+```
```

Comment:
is this correct (and correctly spelled), @tomwwinter ?

I have seen this format somewhere but only found the .env format in production use. So this is just an assumption ...